<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>471</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="page">
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>280</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Begin</string>
         </property>
        </widget>
        <widget class="QLabel" name="mainPic_label">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>0</y>
           <width>291</width>
           <height>181</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QTextBrowser" name="textBrowser">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>200</y>
           <width>311</width>
           <height>71</height>
          </rect>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;College Tour: An indepth application to help those visit the universities they want to see, find crucial information, and purchase any souvenirs they might want along the way. Come join us for an educational trip!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>161</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;College Tour&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QTextBrowser" name="textBrowser_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>40</y>
           <width>201</width>
           <height>271</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>230</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Plan A Trip</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>270</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Quit</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_4">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>290</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Admin</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_3">
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>181</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>College Tour Page (default/custom)</string>
         </property>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>50</y>
           <width>251</width>
           <height>201</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="text">
             <string>Default Trip (UCI)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_6">
            <property name="text">
             <string>Default Trip (ASU)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_7">
            <property name="text">
             <string>Custom Trip</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="backCollegeButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>290</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_4">
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>150</y>
           <width>181</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Admin Panel (index 3)</string>
         </property>
        </widget>
        <widget class="QPushButton" name="backCollegeButton_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>290</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
